src/Services/EncounterService.php

```diff
@@ -340,8 +341,10 @@ class.notes as class_title,
         if ($statementResults) {
             while ($row = sqlFetchArray($statementResults)) {
```

Comment:
after line 44, add the following:
```
        (new UuidRegistry(['table_name' => self::PATIENT_TABLE]))->createMissingUuids();
        (new UuidRegistry(['table_name' => self::PROVIDER_TABLE]))->createMissingUuids();
```

This will ensure that all the elements used by this service are assigned uuid's
Once that is in, then this is ready for the codebase.

Reply:
Made the changes accordingly.